src/_assets/page_assets/tools/android_studio.svg

```diff
@@ -0,0 +1,811 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 18.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->

```

Comment:
This SVG needs to be scrubbed (to remove AI extensions that are unnecessary for use of the SVG itself).

Reply:
I uploaded a new svg, this time around from go/icons